yarn.lock

```diff
@@ -1563,6 +1573,21 @@ __metadata:
   languageName: node
   linkType: hard
 
+"@polkadot/api-augment@npm:9.6.2":
```

Comment:
Apparently I could not get to remove duplicates unless I updated all polkadot dependencies to the latest version, which I did in https://github.com/KILTprotocol/sdk-js/pull/677/commits/4074343193c63dffcd470ccf89a89f429c915179.

Reply:
Good catch, by the way!

Reply:
The only reason for me to look at the diffs of yarn.lock :)

Reply:
By the way, babel duplicates in the new diff seem innocent, however it would be worth it to make sure that the bundle size doesn’t change significantly with new dependencies

Reply:
Is there a fix for it? Neither `yarn --dedupe` not `npx yarn-dedupe` work.

Reply:
Let’s bother ourselves with that only if we have to :)

Reply:
it would be `yarn dedupe` but if that doesn't work, you can also remove anything babel related from the lock file and run `yarn install`

Reply:
Ok `yarn dedupe` this time seems to have removed some duplicates -> https://github.com/KILTprotocol/sdk-js/pull/677/commits/be41879ba04a7c63603df394a1b1653d0d938a1f